src/pages/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>GameWave</title>
+</head>
+<body>
+<link rel="stylesheet" href="../../public/css/output.css">
```

Comment:
any file in the src folder should communicate directly with the public folder. 
The expectation is that the index.html file in src will, at the very least, get copied over automatically in the public folder. And the index.html in the public folder is the one served and seen. And that one has direct access to public folder. Therefore the href would be something like ./css/output.css